test/e2e/systemd_activate_test.go

```diff
@@ -65,42 +64,18 @@ var _ = Describe("Systemd activate", func() {
 		Expect(err).ToNot(HaveOccurred())
 		addr := net.JoinHostPort(host, strconv.Itoa(port))
 
-		// Make a temporary root directory
-		tmpRootDir := filepath.Join(tempDir, "server_root")
-		err = os.Mkdir(tmpRootDir, 0755)
-		Expect(err).ToNot(HaveOccurred())
-		defer os.RemoveAll(tmpRootDir)
-
-		// When SELinux is enabled, a storage root directory should be
-		// labeled with a specific value
-		if selinux.GetEnabled() {
-			rootDir := "/var/lib/containers"
-			label := "container_var_lib_t"
-			if isRootless() {
-				rootDir = filepath.Join(os.Getenv("HOME"), ".local/share/containers")
-				label = "data_home_t"
-			}
-
-			args := []string{"--reference", rootDir, tmpRootDir}
-			// If rootDir doesn't exist, use "chcon -t" to label tmpRootDir
-			// instead of "chcon --reference"
-			if _, err := os.Stat(rootDir); err != nil {
-				args = []string{"-t", label, tmpRootDir}
-			}
-
-			chcon := testUtils.SystemExec("chcon", args)
-			Expect(chcon).Should(Exit(0))
-		}
+		podmanOptions := podmanTest.makeOptions(nil, false, false)
 
-		activateSession := testUtils.StartSystemExec(activate, []string{
+		systemdArgs := []string{
 			"-E", "http_proxy", "-E", "https_proxy", "-E", "no_proxy",
 			"-E", "HTTP_PROXY", "-E", "HTTPS_PROXY", "-E", "NO_PROXY",
+			"-E", "XDG_RUNTIME_DIR=" + os.Getenv("XDG_RUNTIME_DIR"),
```

Comment:
you can just add `-E XDG_RUNTIME_DIR` systemd will automatically read the value.